### PR TITLE
First draft of the new README

### PR DESCRIPTION
**Pull request by Antonin-Carette-RedArtGames**
Based on the discussion we had in https://github.com/adventurebrew/boozook/issues/14, I propose here to rewrite the README and include some new features like the interactive mode.

I included directly in the README some `[TODO _ TO_DISCUSS]` statements, which are, most of time, misunderstanding from my part (how to use the script?) or errors when I tried to use the script.

Please directly comment the README, or add new remarks.
I will check them each morning (except the weekend).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,16 @@
 # boozook
-Tools for extracting and editing Coktel Vision game resources.
```
Comment (BJNFNE) [inline README.md:2]: I think we should keep this Line.

File: README.md [hunk 2]
```diff
@@ -42,29 +42,140 @@ Tools for extracting and editing Coktel Vision game resources.
...
+This will extract, in a folder called `texts` at the root of the `boozook` folder (the `TEXTS_GAME_PATH` folder).
+
+### Text rebuild
+To recompress the modified texts, back to an archive, you can use the `test` script as well  **but** with the `-r` (or `--rebuild`) argument:
+```bash
+python -m boozook.text -r GAME_PATH
+```
+
+> [TODO _ TO_DISCUSS]  
+> How to use the rebuild tool ?
```
Comment (BJNFNE) [inline README.md]: I think we keep the -r command for rebuilding the archives to the patch folder.

File: README.md [hunk 3]
```diff
@@ -42,29 +42,140 @@ Tools for extracting and editing Coktel Vision game resources.
...
+
+### Text rebuild
+To recompress the modified texts, back to an archive, you can use the `test` script as well  **but** with the `-r` (or `--rebuild`) argument:
+```bash
+python -m boozook.text -r GAME_PATH
+```
+
+> [TODO _ TO_DISCUSS]  
+> How to use the rebuild tool ?  
+> Do we have to point to the texts output directory?
```
Comment (BJNFNE) [inline README.md]: we should put the extracted text to the texts folder as before.

**Review by BJNFNE — COMMENTED** [review]
Looks good so far. @BLooperZ any objections?

**Comment by BLooperZ** [comment]
Thank you very much for this amzing work.
I have a few notes I would like to address though:

1. It is preferable to use the runner script for everything,
rather than usung individual scripts

so if someone would like to extract graphics the command is
```boozook --graphics GAME_PATH```
and to extract both fonts and graphics
```boozook --graphics --fonts GAME_PATH```

2. The interactive mode is not experimental actually, and is the recommended way to use the tool, so we should probably also suggest it first in the readme

The experimental flag is only for having the option to decompile scripts, both in interactive mode and script usage.

so ```boozok --scripts *.TOT GAME_PATH`` won't work
but ```boozook --experimental --scripts *.TOT GAME_PATH``` will.

3. as you can note from my example, we can use ```boozook``` rather than ```python -m boozook.runner```

File: README.md (hunk 1 above)
Comment (BLooperZ) [reply README.md:2]: I agree

**Comment by Antonin-Carette-RedArtGames** [comment]
Thank you for the comments!

I will update the README and submit the changes for tomorrow morning :)

**Comment by Antonin-Carette-RedArtGames** [comment]
I reworked the README based on your comments.
Please let me know if you want changes or additions :)

File: README.md (hunk 1 above)
Comment (Antonin-Carette-RedArtGames) [reply README.md:2]: I put back the old description.

File: README.md (hunk 2 above)
Comment (Antonin-Carette-RedArtGames) [reply README.md]: Done!
And I removed the TODO statement

File: README.md (hunk 3 above)
Comment (Antonin-Carette-RedArtGames) [reply README.md]: Done!
And I removed the TODO statement

**Comment by BJNFNE** [comment]
> I reworked the README based on your comments.
> 
> Please let me know if you want changes or additions :)

Looks good, @BLooperZ any objections? If not we can merge it.

**Comment by BLooperZ** [comment]
Thank you, this is very helpful to understand what information should be covered.
That said, there are few instructions that are still incorrect, so I can't accept it yet, althoug it gives me an idea of structuring the readme a bit differently.
I'll try writing it in the other structure and update soon, I would like to hear your opinion about it if it's okay.
Thanks again

**Comment by Antonin-Carette-RedArtGames** [comment]
Sure, no problem, take your time :)

**Comment by BJNFNE** [comment]
What is the status of this PR?

**Comment by Antonin-Carette-RedArtGames** [comment]
JFYI, I will leave my current company end of December, so I might not be available for reviews & new contributions starting mid-December (at least under this account).

**Comment by BLooperZ** [comment]
Thank you, closing in favor of #19

**Pull request closed without merging**